scripts/travis/test.sh

```diff
@@ -12,5 +12,5 @@ if [ "${OS}-${ARCH}" = "linux-arm" ]; then
 fi
 
 make fixcheck
-if [[ "${OS}" != "darwin" ]]; then scripts/travis/run_tests.sh; fi
-if [[ "${OS}" != "darwin" ]]; then scripts/travis/after_build.sh; fi
+scripts/travis/run_tests.sh;
```

Comment:
this would enable for e2e as well as unit test on Mac. Since you meant to enable just e2e on this change, I'd suggest you'll be more specific. see run_test.sh for details.